roles/keystone/tasks/monitoring.yml

```diff
@@ -3,8 +3,16 @@
   sensu_process_check: service=keystone-all state=absent
   notify: restart sensu-client
 
-- name: keystone process check
+- name: remove simple keystone process check
   sensu_process_check: service=uwsgi
+    state: absent
+  notify: restart sensu-client
+
+- name: keystone process check
+  sensu_check:
+    name: nova_placement_process
```

Comment:
use sensu_check here is reasonable, but this sensu_check name should be changed.